### DISCUSSION TITLE
Валидация форм ⚡️ Angular с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/angdev.ru#29] [Announcements]
# Валидация форм ⚡️ Angular с примерами кода

Независимо от выбранной технологии построения формы ее валидация происходит на основе данных о состоянии самой формы и каждого ее поля в отдельности

https://angdev.ru/doc/forms-validation/

**Comment by Argentum4k** [comment]
Кажется, в примере check-goods-left.validator.ts должен быть оператор "map" вместо "tap"
